### PR TITLE
Added a new required info

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Problem description
Done:
- Added a new required info. 
- Changed the way how the required messages are rendered. 
- Updated old required info messages.

### CODE REVIEW

**Review by MajoBerger — APPROVED** [review]

File: src/assets/i18n/en.json5 [hunk 1]
```diff
@@ -692,6 +692,26 @@
...
+
+  "clarin.license.agreement.signer.EXTRA_EMAIL": "E-mail",
+
+  "clarin.license.agreement.signer.ORGANIZATION": "Organization",
+
+  "clarin.license.agreement.signer.REQUIRED_ORGANIZATION": "Organization",
+
+  "clarin.license.agreement.signer.INTENDED_USE": "Intended use",
+
+  "clarin.license.agreement.signer.ACA_ORG_NAME_AND_SEAT": "Name and seat (address) of your academic institution",
```
Comment (vidiecan) [inline src/assets/i18n/en.json5:713]: seat?

File: src/assets/i18n/cs.json5 [hunk 2]
```diff
@@ -667,6 +667,26 @@
...
+  // "clarin.license.agreement.signer.EXTRA_EMAIL": "E-mail",
+  "clarin.license.agreement.signer.EXTRA_EMAIL": "E-mail",
+  // "clarin.license.agreement.signer.ORGANIZATION": "Organization",
+  "clarin.license.agreement.signer.ORGANIZATION": "Organizace",
+  // "clarin.license.agreement.signer.REQUIRED_ORGANIZATION": "Organization",
+  "clarin.license.agreement.signer.REQUIRED_ORGANIZATION": "Organizace",
+  // "clarin.license.agreement.signer.INTENDED_USE": "Intended use",
+  "clarin.license.agreement.signer.INTENDED_USE": "Zamýšlené použití",
+  // "clarin.license.agreement.signer.ACA_ORG_NAME_AND_SEAT": "Name and seat (address) of your academic institution",
+  "clarin.license.agreement.signer.ACA_ORG_NAME_AND_SEAT": "Název a sídlo vědecké instituce",
```
Comment (vidiecan) [inline src/assets/i18n/cs.json5:689]: akademicke

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: src/assets/i18n/cs.json5 [hunk 3]
```diff
@@ -656,7 +656,7 @@
   // "clarin.license.agreement.signer.header.info": ["The information below identifies you, the SIGNER. If the information is incorrect, please contact our", "Help Desk.", "This information will be stored as part of the electronic signatures."],
   "clarin.license.agreement.signer.header.info": ["Informace uvedené níže vás, PODEPSANÉHO, identifikují. Pokud jsou tyto informace nesprávné, kontaktujte prosím", "poradnu.", "Tyto informace budou uloženy spolu se záznamem o souhlasu."],
   // "clarin.license.agreement.item.handle": "Item handle",
-  "clarin.license.agreement.item.handle": "Klika položky",
+  "clarin.license.agreement.item.handle": "Handle položky",
```
Comment (vidiecan) [inline src/assets/i18n/cs.json5:659]: PID? Identifikator? Musi to byt vzdy Handle?

File: src/assets/i18n/en.json5 (hunk 1 above)
Comment (MajoBerger) [reply src/assets/i18n/en.json5:713]: https://www.lawinsider.com/dictionary/seat-of-the-institution

Comment (milanmajchrak) [reply src/assets/i18n/en.json5:713]: I just copied it from the UFAL: https://github.com/ufal/clarin-dspace/blob/clarin-dev/dspace/modules/xmlui/src/main/webapp/i18n/messages_cs.xml#L1972

File: src/assets/i18n/cs.json5 (hunk 2 above)
Comment (milanmajchrak) [reply src/assets/i18n/cs.json5:689]: I just copied it from the UFAL: https://github.com/ufal/clarin-dspace/blob/clarin-dev/dspace/modules/xmlui/src/main/webapp/i18n/messages_cs.xml#L1972

File: src/assets/i18n/cs.json5 (hunk 3 above)
Comment (milanmajchrak) [reply src/assets/i18n/cs.json5:659]: Zatiaľ tam zobrazujeme vždy Handle, ak by to mohol byť aj DOI, tak by trebalo trochu pomenit veci. Každý Item ma vždy nejaký Handle, tak ideálna oprava by bola, ak by zobrazilo aj DOI popri Handle, ak ma Item DOI.

**Review by vidiecan — APPROVED** [review]

**Pull request merged**